card.py

```diff
@@ -1,7 +1,7 @@
 from constants import *
 class Card:
-    def __init__(self, _type, value):
-        self.type = _type
+    def __init__(self, type, value):
```

Comment:
well, it works, but here's the reason I don't recommend using `type` https://stackoverflow.com/a/10568115

Reply:
Changed type to suit everywhere